HISTORY.md

```diff
@@ -34,6 +34,7 @@
 * Allow open array stats to be printed without read query [#2131](https://github.com/TileDB-Inc/TileDB/pull/2131)
 
 ## Bug fixes
+* Fixes a potential crash when retrying incomplete reads [#2137](https://github.com/TileDB-Inc/TileDB/pull/2137)
```

Comment:
This should go in the pending section nor a new 2.2.6 sectiy.

Reply:
Done, ty